DXMainClient/Online/Connection.cs

```diff
@@ -618,6 +619,10 @@ private void PerformCommand(string message)
                         case 311: // Reply to WHOIS NAME query
                             connectionManager.OnWhoReplyReceived(parameters[2], parameters[3], parameters[1], string.Empty);
                             break;
+                        case 315: // End of WHO query (RPL_ENDOFWHO)
+                            connectionManager.OnWhoQueryComplete(parameters[1], new List<(string ident, string host, string userName, string extraInfo)>(whoResponseList));
```

Comment:
Sorry, another review comment.

I would prefer `OnWhoQueryComplete()` returns a list using an out keyword, instead of creating a `whoResponseList` in advance.